src/layouts/partials.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { ReactElement, ReactNode } from 'react';
+
+interface IHeaderProperties {
+  readonly assetPath?: string;
+  readonly location: string;
+  readonly isPlatformAdmin: boolean;
+}
+
+interface IPhaseBannerProperties {
+  readonly tag: {
+    readonly text: string;
+  };
+  readonly children?: ReactNode;
+}
+
+interface IMainProperties {
+  readonly classess?: ReadonlyArray<string>;
+
+  readonly contentStart?: ReactElement;
+  readonly contentEnd?: ReactElement;
+  readonly language?: string;
+
+  readonly children: ReactElement;
+}
+
+interface ISubNavigationItem {
+  readonly text: string;
+  readonly link: string;
+  readonly active?: boolean;
+}
+
+export interface ISubNavigationProperties {
+  readonly title?: string;
+  readonly items: ReadonlyArray<ISubNavigationItem>;
+}
+
+export function Header(params: IHeaderProperties): ReactElement {
+  const platformLink = (
+    <li className="govuk-header__navigation-item admin">
+      <a className="govuk-header__link" href="/platform-admin">
+        Platform admin
+      </a>
+    </li>
+  );
+
+  const assetPath = params.assetPath || '/assets';
+
+  return (
+    <header className="govuk-header" role="banner" data-module="govuk-header">
+      <div className="govuk-header__container govuk-width-container">
+        <div className="govuk-header__logo">
+          <a href="#" className="govuk-header__link govuk-header__link--homepage">
```

Comment:
I'm curious if the `href="#"` is intentional?

Reply:
Good catch. I'd much rather have `/` there, which would redirect to admin home page... This must be an artefact of me copying design system fresh to make sure the element is roughly up to date.